tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -35,6 +35,76 @@ class TargetedSuite extends SemanticdbSuite {
     }
   )
 
+  occurrences(
```

Comment:
Can we move this test to `ExpectSuite`? Similar to https://github.com/scalameta/scalameta/blob/2dbe2d93096d78d1670fb7cb939108cb740b0bcd/semanticdb/integration/src/main/scala/example/Issue1749.scala

You can run `save-expect` to update the generated output files.

`TargetedSuite` is primarily used to provide a fast edit/test feedback while iterating on changes but it doesn't test as comprehensively as `ExpectSuite`